.env.docker

```
@@ -10,7 +10,7 @@ VITE_COGNITO_REGION=us-east-1
 # Puppeteer backend
 PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium
 LINKEDIN_BASE_URL=http://mock-linkedin:3333
-DYNAMODB_TABLE_NAME=linkedin-advanced-search-test
+DYNAMODB_TABLE_NAME=warmreach-test
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DYNAMODB_TABLE_NAME key should go before the LINKEDIN_BASE_URL key

(UnorderedKey)